schema.go

```diff
@@ -278,11 +278,11 @@ func (s *Schema) validate(v interface{}) error {
 			var missing []string
 			for _, pname := range s.Required {
 				if _, ok := v[pname]; !ok {
-					missing = append(missing, strconv.Quote(pname))
```

Comment:
pname may contain comma or space. because of this it was quoted in message

Reply:
We want the raw JSON pointers here (e.g. `#/foo/bar`) because we want to pass that to the error details. The error message itself is still being quoted: https://github.com/ory/jsonschema/pull/1/files#diff-02934e0fdc85f35aa62e19cf7c6f6bcaR39

Reply:
got it. but before including pname in json pointer, some special characters needs to be escaped.
check https://github.com/santhosh-tekuri/jsonschema/blob/master/schema.go#L557

Reply:
Right! I though [`joinPtr`](https://github.com/ory/jsonschema/pull/1/files#diff-02934e0fdc85f35aa62e19cf7c6f6bcaR21) would do that but I assumed that incorrectly.